.npmignore

```diff
@@ -0,0 +1 @@
+node_modules/
```

Comment:
@thypirate Please add a new line to the end of this file

Reply:
@luis737 can we let this go or can you squash a commit with the change so we can ship this?